userguide/account.rst

```diff
@@ -83,31 +83,31 @@ summarizes the available options when creating a group.
    |                     |           |                                                                                                                          |
    +=====================+===========+==========================================================================================================================+
-   | Group ID            | string    | the next available group ID will be suggested for you; by convention, UNIX groups containing user accounts have          |
-   |                     |           | an ID greater than 1000 and groups required by a service have an ID equal to the default port number used by the         |
-   |                     |           | service (e.g. the sshd group has an ID of 22)                                                                            |
+   | GID                 | string    | The next available group ID is suggested. By convention, UNIX groups containing user accounts have an ID greater than    |
+   |                     |           | **1000** and groups required by a service have an ID equal to the default port number used by the service. For example,  |
+   |                     |           | the :literal:`sshd` group has an ID of **22**.                                                                           |
    |                     |           |                                                                                                                          |
    +---------------------+-----------+--------------------------------------------------------------------------------------------------------------------------+
-   | Group Name          | string    | mandatory                                                                                                                |
+   | Name                | string    | Enter a name for the new group. This is required.                                                                        |
    |                     |           |                                                                                                                          |
    +---------------------+-----------+--------------------------------------------------------------------------------------------------------------------------+
-   | Permit Sudo         | checkbox  | if checked, members of the group have permission to use `sudo <http://www.sudo.ws/>`_; when using sudo, a user will      |
-   |                     |           | be prompted for their own password                                                                                       |
+   | Permit Sudo         | checkbox  | Check this to allow members of the group to use `sudo <http://www.sudo.ws/>`__. When using :command:`sudo`, a user is    |
+   |                     |           | prompted for their own password.                                                                                         |
```

Comment:
Looks like trailing whitespace added here.

Reply:
That one is part of the table - I can't find any whitespace errors apart from the usual table formatting.